_posts/2020-11-2-kubernetes-webhooks-made-easy-with-openfaas-part-2.md

```diff
@@ -0,0 +1,291 @@
+---
+title: "Kubernetes Webhooks made easy with OpenFaaS - Part 2"
+description: "In this post you'll learn how to write Kubernetes Admission webhooks using OpenFaaS functions - Part 2"
+date: 2020-11-2
+image: /images/2020-10-27-k8s-validatingwebhook-openfaas/puzzle.jpg
+categories:
+  - arkade
+  - kubectl
+  - faas-cli
+  - admissionwebhooks
+  - mutatingadmissionwebhooks
+  - k8s extensibility
+author_staff_member: developer-guy
+dark_background: true
+---
+
+In this post you'll learn how to write Kubernetes Admission webhooks using OpenFaaS functions - Part 2
```

Comment:
`Learn how to write your Kubernetes Admission Controllers with OpenFaaS functions` <- same as the description please

Reply:
Done